src/bionics_ui.cpp

```diff
@@ -411,22 +502,10 @@ static nc_color get_bionic_text_color( const bionic &bio, const bool isHighlight
     return type;
 }
 
-static std::vector<bionic *> filtered_bionics( bionic_collection &all_bionics,
-        bionic_tab_mode mode )
-{
-    std::vector< bionic *>filtered_entries;
-    for( auto &elem : all_bionics ) {
-        if( ( mode == TAB_ACTIVE ) == elem.id->activated ) {
-            filtered_entries.push_back( &elem );
-        }
-    }
-    return filtered_entries;
-}
-
 void player::power_bionics()
 {
-    std::vector <bionic *> passive = filtered_bionics( *my_bionics, TAB_PASSIVE );
-    std::vector <bionic *> active = filtered_bionics( *my_bionics, TAB_ACTIVE );
+    ::sorted_bionics passive = ::filtered_bionics( *my_bionics, TAB_PASSIVE );
+    ::sorted_bionics active = ::filtered_bionics( *my_bionics, TAB_ACTIVE );
```

Comment:
```suggestion
    sorted_bionics passive = filtered_bionics( *my_bionics, TAB_PASSIVE );
    sorted_bionics active = filtered_bionics( *my_bionics, TAB_ACTIVE );
```
Normally no need to explicitly specify that you're using things from the global namespace (there are more changes like this).

Reply:
Those are for clarity only. I can change them